lib/workers/repository/reconfigure/index.spec.ts

```diff
@@ -26,6 +27,9 @@ describe('workers/repository/reconfigure/index', () => {
   const config: RenovateConfig = {
     branchPrefix: 'prefix/',
     baseBranch: 'base',
+    statusCheckNames: partial<RenovateConfig['statusCheckNames']>({
```

Comment:
are you sure you need the partial here?

Reply:
Yes. Normally this object will have all the 4 status check strings.

Reply:
Sure, but you are using a `Record` which isn't enforcing that all keys are defined.